### PR TITLE
Move EIP-2098 to final call

### PR DESCRIPTION
**Pull request by ricmoo**
Moving EIP-2098 to final call.

### CODE REVIEW

File: EIPS/eip-2098.md [hunk 1]
```diff
@@ -1,18 +1,15 @@
 ---
 eip: 2098
 title: Compact Signature Representation
-status: Review
+description: A compact representation of an Ethereum Signature.
+status: Final Call
```
Comment (poojaranjan) [inline EIPS/eip-2098.md:5]:
```suggestion
status: Last Call
last-call-deadline: 2022-01-11
```

**Review by lightclient — CHANGES_REQUESTED** [review]
* `s/Test Vectors/Test Cases`
* `Gas Analysis` is not a valid `##` section per EIP-1, please include that information in one of the other sections. `Rationale` seems like the best choice?
* `s/Implementations/Reference Implementation`
* `Acknowledgments` is not a valid EIP section, please remove
* `Security Considerations` is missing

**Review by Pencab — APPROVED** [review]

**Comment by ricmoo** [comment]
These changes have been made.

File: EIPS/eip-2098.md [hunk 2]
```diff
@@ -1,18 +1,15 @@
 ---
 eip: 2098
 title: Compact Signature Representation
-status: Review
+description: A compact representation of an Ethereum Signature.
+status: Last Call
```
Comment (MicahZoltu) [inline EIPS/eip-2098.md:5]:
```suggestion
status: Last Call
last-call-deadline: 2022-02-13
```

File: EIPS/eip-2098.md [hunk 3]
```diff
@@ -122,100 +120,17 @@ Compact Signature:
...
-Compact:
-  Hash:     0x513bd8bd8710a84903235ca60591fc60f2f3db524d14cc1b6874edc628512176
-  Data:     0x1230abb66ac9e083f46825a57816e023ae641ee45ac9b82cc2c370c10ec1ef4c835fa182e95b1a8633ee7ff851f68cf4303030b1e2596d686cafd9803cef74919a9139291c492d05696da754cf342bec961d00d9f7dfac3ab90b1e9352177c7b9bfa2a5d
-  Length:   100
-  Gas Used: 37329
-```
-
-
-## Implementations
+## Reference Implementations
```
Comment (lightclient) [inline EIPS/eip-2098.md:123]:
```suggestion
## Reference Implementation
```

**Review by lightclient — CHANGES_REQUESTED** [review]

**Review by lightclient — APPROVED** [review]

**Pull request merged**